src/components/CandidateTaskView/CandidateTaskView.js

```diff
@@ -5,14 +5,19 @@ import "./CandidateTaskView.styl";
 import { getRoot } from "mobx-state-tree";
 import { useEffect, useState } from "react";
 import { format } from "date-fns";
+import { FF_LSDV_4711, isFF } from "../../utils/feature-flags";
+
+const imgDefaultProps = {};
```

Comment:
it's better to uppercase top-level constants

```suggestion
const IMG_DEFAULT_PROPS = {};
```